### DISCUSSION TITLE
支持加载网络路径的pag文件

### DISCUSSION THREAD

**Discussion by yuximin** [Tencent/libpag#1818] [Q&A] [answered]
![screenshot-20231101-155053](https://github.com/Tencent/libpag/assets/37434036/a7d4fd39-5bb4-44fc-b4ea-ae00a32e82c9)
你好，我在版本日志中看到 "新增能力 支持加载网络路径的 pag 文件"，但是我在文档中没有找到相关的API。请问这个功能应该如何使用？

**Comment by kevingpqi123** [comment, marked as answer]
PAGView、PAGImageView 以及 PAGFile setPath 有同步和异步的接口都可以使用
